common/src/main/java/io/novafoundation/nova/common/data/network/runtime/binding/AccountInfo.kt

```diff
@@ -62,6 +68,9 @@ fun bindAccountInfo(scale: String, runtime: RuntimeSnapshot): AccountInfo {
     val dynamicInstance = type.fromHexOrNull(runtime, scale).cast<Struct.Instance>()
 
     return AccountInfo(
+        consumers = dynamicInstance.getTyped("consumers"),
+        providers = dynamicInstance.getTyped("providers"),
+        sufficients = dynamicInstance.getTyped("sufficients"),
```

Comment:
We should take into account older versions of AccountInfo, which had different set of fields:
very first version `refCount: u8`
then `refCount: u32`
then `providers: u32, consumers: u32`

I think we can handle it quite easily by using using dynamicInstance.getTyped<Balance?>("consumers").orZero() and the same for the rest of the fields
You can check that everything works fine by checking error logs for errors related to balance updates since AccountInfo is used in balances as well